app/views/schools/cohorts/show.html.erb

```diff
@@ -0,0 +1,64 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-3"><%= @cohort.start_year %></h1>
+
+    <p class="govuk-body-l govuk-!-margin-bottom-8">
+    <% if @school_cohort[:induction_programme_choice] === "core_induction_programme" %>
```

Comment:
`@school_cohort.induction_programme_choice` should work too - I dislike hash notation in views :P

Reply:
me too i didn't even know that was an option